apps/vr-tests-react-components/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
   "rules": {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26943&commitId=338efbaaddb8af4e7dcee8bf81f48c828486597d&env=prod)

✅ There was 6 screenshots added, 0 screenshots removed, 1857 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FluentProvider CustomStyleHooks.Button.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.Default.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>